Add more straightforward README example and general clean-up

cdrage opened the pull request:
Moved the example to the top as a "use case".

Added more widgets.

Cleaned up some other parts of the README (shell autocompletion, etc.)
in order to avoid repetition in README.

kadel commented on README.md:
```diff
@@ -1,24 +1,55 @@
+  containers:
+  - image: nginx
+    ports:
+    - port: 8080:80
```
nginx on docker hub is listenning on port 80.

This should be `80:8080` or just `80`

cdrage replied:
Shouldn't it still be `8080:80`?

Ex: ` docker run --name some-nginx -d -p 8080:80 some-content-nginx` ?

EDIT: nvm, I'm an idiot.

kadel replied:
yeh, It can be bit confusing :-( if you come from docker/docker-compose. OpenCompose has a different order for ports. But there is a good reason for it ;-)

cdrage committed:
Add more straightforward README example and general clean-up

Moved the example to the top as a "use case".

Added more widgets.

Cleaned up some other parts of the README (shell autocompletion, etc.)
in order to avoid repetition in README.

cdrage commented:
@kadel Updated!
